### ISSUE TITLE
JoinTable missing

### ISSUE THREAD

**Issue by ankurhanda** [clcarwin/convert_torch_to_pytorch#36]
Can you also add JoinTable please? https://github.com/pytorch/pytorch/blob/master/torch/legacy/nn/JoinTable.py

**Comment by r2d3** [comment]
JoinTable could be implemented by

in `lua_recursive_source`
~~~
elif name == 'JoinTable':
    s += ['LambdaReduce(lambda x,y: torch.cat((x,y),dim={}), # JoinTable'.format(m.dimension-1)]
~~~

and something similar in `lua_recursive_model`

~~~
elif name == 'JoinTable':
    dim = m.dimension-1
    n = LambdaReduce(lambda x,y,dim=dim: torch.cat((x,y),dim))
    add_submodule(seq,n)
~~~
